src/utilities/__tests__/buildClientSchema-test.js

```diff
@@ -29,10 +29,10 @@ import { introspectionFromSchema } from '../introspectionFromSchema';
 /**
  * This function does a full cycle of going from a string with the contents of
  * the SDL, build in-memory GraphQLSchema from it, produce a client-side
- * representation of the schema by using "buildClientSchema"and then finally
- * printing that that schema into the SDL
+ * representation of the schema by using "buildClientSchema" and then
+ * returns that schema printed as SDL.
```

Comment:
Not a big deal: but for future PRs it would be great to keep grammar fixes as separate PR.